qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/parser/template/scanner/TemplateScanner.java

```diff
@@ -255,6 +257,10 @@ protected TokenType internalScan() {
 		finishToken(offset, TokenType.Unknown, errorMessage);
 	}
 
+	private boolean isValidIdentifierStart(int ch) {
+		return Character.isDigit(ch) || Character.isAlphabetic(ch) || ch == '_' || ch == '}';

```

Comment:
looks weird to say } is a valid identifier start character

Reply:
Qute code use this name https://github.com/quarkusio/quarkus/blob/4940f063022e3fc8fc9dd723d1b58f4ca4b2c4d3/independent-projects/qute/core/src/main/java/io/quarkus/qute/Parser.java#L309

Reply:
yes, and } is not in there

```java
// A valid identifier must start with a digit, alphabet, underscore, comment delimiter, cdata start delimiter or a tag command (e.g. # for sections)
        return Tag.isCommand(character) || character == COMMENT_DELIMITER || character == CDATA_START_DELIMITER
                || character == CDATA_START_DELIMITER_OLD
                || character == UNDERSCORE
                || Character.isDigit(character)
                || Character.isAlphabetic(character);
```

Reply:
Good catch! I fixed and write a test for that.